modules/heat_conduction/include/materials/GapConductance.h

```diff
@@ -88,6 +88,7 @@ class GapConductance : public Material
   const VariableValue & _gap_temp_value;
   MaterialProperty<Real> & _gap_conductance;
   MaterialProperty<Real> & _gap_conductance_dT;
+  MaterialProperty<Real> & _gap_conductivity_property;
```

Comment:
Minor: Do you really want the suffix `_propery` on there? We don't typically do this.